cli.ts

```diff
@@ -294,6 +319,66 @@ const initProject = (projectDir: SanitizedAbsPath, configDir: SanitizedPath, i18
     map (() => i18n.__("bootstrapMsg"))
 )
 
+const scaffoldProject = (scaffoldUrl: string, projectDir: SanitizedAbsPath, configDir: SanitizedPath, i18n: i18n, maxConcurrency: number, quickstart: string) => pipe(
+    // Clone git into destination folder (Initial version assumes git is installed)
+    attemptP(async () => {
+        // TODO: Convert to fluture
+        // TODO: i18n of messages
+
+        console.log(`scaffolding\n into: ${projectDir.value}\n from: ${scaffoldUrl}\n...`)
+
+        // Verify projectDir does not exist
+        try {
+            await Deno.stat(projectDir.value);
+            return Promise.reject({kind: 'E_SCAFFOLD_PROJECT_DIR_ALREADY_EXISTS', msg: 'TODO i18n message'})
+        } catch(_e) {
+            // Expect exception when trying to stat a new directory
+        }
+
+        // git clone
+        console.log(`git clone...`)
+
+        const cloneProcess = Deno.run({
+            cmd: ["git", "clone", scaffoldUrl, projectDir.value],
+        });
+        const cloneResult = await cloneProcess.status();
+        
+        if (!cloneResult.success) {
+            return Promise.reject({kind: 'E_SCAFFOLD_URL_GIT_CLONE_FAILED', msg: 'TODO i18n message'})
+        }
+
+
+        // Run init found in .taq/scaffold.json
+        console.log(`initializing...`)
+        // TODO: Run initialization script
+        // Load .taq/scaffold.json (if it exists)
+        // Run init command
+
+
+        // Cleanup:
+        console.log(`cleanup...`)
+
+        try {
```

Comment:
Another good example of a reusable utility here. Would you be up for adding a `rm` utility to taqueria-utils with a signature something like:
`string => Future<E_RM, SanitizedAbsPath>` ?